okta/provider.go

```diff
@@ -160,6 +162,7 @@ func Provider() terraform.ResourceProvider {
 			"okta_saml_idp_signing_key":      deprecateIncorrectNaming(resourceIdpSigningKey(), idpSamlKey),
 			"okta_social_idp":                deprecateIncorrectNaming(resourceIdpSocial(), idpSocial),
 			"okta_bookmark_app":              deprecateIncorrectNaming(resourceAppBookmark(), appBookmark),
+			"okta_basic_auth_app":            deprecateIncorrectNaming(resourceAppBasicAuth(), appBasicAuth),
```

Comment:
You can drop this line. Some resources were incorrectly named and thus we are still supported them with a deprecation message. All new resources can avoid this.